### ISSUE TITLE
Wrong serial number in dmidecode on APU1 

### ISSUE THREAD

**Issue by lwcislo** [pcengines/coreboot#402]
coreboot image : apu1_v4.11.0.6.rom
platform: APU1

Dmidecode returns an improper Serial Number:
```
Handle 0x0002, DMI type 2, 14 bytes
Base Board Information
        Manufacturer: PC Engines
        Product Name: apu1
        Version: 1.0
        Serial Number: -64
        Asset Tag: Not Specified
        Features: None
        Location In Chassis: Not Specified
        Chassis Handle: 0x0003
        Type: Unknown
```

**Comment by miczyg1** [comment]
FIxed in v4.12.0.2

**Comment by pcenginetest** [comment]
Back in v4.12.03
